server/src/main/java/jass/server/util/GameUtil.java

```diff
@@ -332,6 +363,246 @@ public void onPlayedCard(final PlayCardData data) throws InterruptedException {
         }
     }
 
+    ////////////////////////////////////////////////////////////////////////////
+    // Figure out the points for cards.
+    ////////////////////////////////////////////////////////////////////////////
+
+    /**
+     * @param cards All cards
+     *
+     * @return Returns the points of all cards together.
+     *
+     * @author Manuele Vaccari
+     */
+    private static int calculateCardPointsTrumpf(final CardEntity... cards) {
+        int points = 0;
+        for (CardEntity card : cards) {
+            points += card.getRank().getPointsTrumpf();
```

Comment:
this only calulcates the sum for normal cards in a trumpf game. You need to catch the case when  jack or nine trump is played.